ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -347,12 +367,13 @@ private static void putEdges(
     int heightIndex = IS_JELLY_BEAN_OR_LATER ? photos.getColumnIndex(Images.Media.HEIGHT) : -1;
     int longitudeIndex = photos.getColumnIndex(Images.Media.LONGITUDE);
     int latitudeIndex = photos.getColumnIndex(Images.Media.LATITUDE);
+    int dataIndex = photos.getColumnIndex(MediaStore.MediaColumns.DATA);
```

Comment:
This variable should not be called `photos` anymore since it's either photos or vides, correct? How about `media`?